### ISSUE TITLE
Hi,I am new player at deep learning, I just want to know,why need set grad=false here?

### ISSUE THREAD

**Issue by Luo-Zhongwei** [DataDistillation/DataDAM#10]
''' Train synthetic data '''
            net = get_network(args.model, channel, num_classes, im_size).to(args.device) # get a random model
            net.train()
            '''----------------------'''
            '''LZW question 3???? WHY HERE
            for param in list(net.parameters()):
                param.requires_grad = False
                '''
            '''----------------------'''

**Comment by DataDistillation** [comment]
Hello, as stated in the paper, our method does not use bi-level optimization; hence, we set our model gradient tracking to zero, This is meant to ensure that the model does not accumulate gradients and slow down our distillation. We would like to refer you to Distribution Matching work, which applies the same technique: https://github.com/VICO-UoE/DatasetCondensation/blob/53f63a672eab9ade02aabe6432706ef56f317d64/main_DM.py#L141
